### ISSUE TITLE
Support packet padding

### ISSUE THREAD

**Issue by kixelated** [pion/rtp#6]
You can set the `Padding` header bit, but there's no way to control the padding size and actually Marshal the padding. Same with Unmarshal.

**Comment by kixelated** [comment]
Also this is a reason why the `srtp` package should operate on `*rtp.Packet`, not `(rtp.Header, []byte)`.

**Comment by maxhawkins** [comment]
Maybe we can add a `Writer` with a `PaddingSize` field kind of like `csv.Writer` from the stdlib and its `UseCRLF` field:

https://godoc.org/encoding/csv#Writer

**Comment by aler9** [comment]
Hello all, this issue prevents RTP packets from being marshaled correctly and makes GStreamer crash. For instance, this:

```go
rtp.Packet{
	Header: rtp.Header{
		Version: 2,
		Padding: true,
	},
	Payload: []byte{0x01, 0x02, 0x03},
}
```

is unmarshaled into a packet with the padding flag and without padding:

```go
[160 0 0 0 0 0 0 0 0 0 0 0 1 2 3]
```

But it should be marshaled into a packet without the padding flag:

```go
[128 0 0 0 0 0 0 0 0 0 0 0 1 2 3]
```

Or it should be marshaled into a packet with the padding flag and padding, whose size that can't be set through pion/rtp:

```go
[160 0 0 0 0 0 0 0 0 0 0 0 1 2 3 0 0 0 4]
```

**Comment by aler9** [comment]
Just realized that this issue is fixed by #155. It can be closed.

**Comment by Sean-Der** [comment]
Oh nice! Thanks @aler9 :)
